.github/workflows/main.yml

```diff
@@ -0,0 +1,74 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Build macOS and Windows
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ adrian/release ]
+#      tags:
+#        - '[0-9]+.[0-9]+.[0-9]+*'
+#        - 'v[0-9]+.[0-9]+.[0-9]+*'
+  pull_request:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  release:
+    name: Release ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - os: macos-10.15
+            profile: release+mac
+            target: x86_64-apple-darwin
+          - os: windows-2019
+            profile: release+windows
+            target: x86_64-pc-windows-msvc
+
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+
+      - name: Setup cargo make
+        uses: davidB/rust-cargo-make@v1
+
+      - name: Get tag
+        id: get_tag
+        run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+
+      - name: Install stable Rust
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable-${{ matrix.target }}
+          profile: minimal
+
+      - name: Build
+        shell: bash
+        run: |
+          cargo make --profile ${{ matrix.profile }} build
+
+#      - name: Package Release
+#        shell: bash
+#        run: |
+#          touch LICENSE.txt
+#          cargo make zip-release-binary-for-target \
```

Comment:
using https://github.com/sagiegurari/cargo-make/blob/master/docs/cargo_make_task_list.md
but it gets skipped locally - could just upload the esthri file?

Reply:
You could try to figure out why its getting skipped (suspect its something to do with the multi-crate setup) with `cargo make --verbose zip-release-binary-for-target` or maybe, yeah, we could just add our own commands to build + zip